### PR TITLE
Fixes species being referenced instead of copied when cloning

### LINKED ISSUE #30817: Lavaland Treasure - Strange Elixir & Cloning
Round: 76167

As a miner, got wings from the strange elixir in lavaland. Died, got cloned. Cloned body still had the wings, but no button in the top left corner to activate or deactivate flight. Gouty spawned in another elixir to fix it, by it had no effect. Had to resort to making a new mob for me to control, along with another elixir.

So in short, wings don't carry over properly when cloned.

### PR DESCRIPTION
**Pull request by XDTM**
Fixes #30817 
Probably hasn't happened yet, but technically the old and new bodies share ability cooldowns and species vars. This fixes that.

Slimepeople still keep their body pool and abductors keep their team status after cloning.

Also fixes broken slime icons.

### CODE REVIEW

**Comment by DoctoreMad** [comment]
>Slimepeople still keep their body pool and abductors keep their team status after cloning.

Wait so if you die at around 42 essence or something you die for real?

File: code/modules/mob/living/carbon/human/species_types/abductors.dm [hunk 1]
```diff
@@ -7,3 +7,7 @@
 	mutanttongue = /obj/item/organ/tongue/abductor
 	var/scientist = 0 // vars to not pollute spieces list with castes
 	var/team = 1
+
+/datum/species/abductor/copy_properties_from(datum/species/old_species)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/abductors.dm]: datum/species/abductor/old_species

File: code/modules/mob/living/carbon/human/species_types/jellypeople.dm [hunk 2]
```diff
@@ -139,6 +140,10 @@
 
 	C.faction |= "slime"
 
+//If you're cloned you get your body pool back
+/datum/species/jelly/slime/copy_properties_from(datum/species/old_species)
```
Comment (Cyberboss) [inline code/modules/mob/living/carbon/human/species_types/jellypeople.dm]: datum/species/jelly/slime/old_species

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by XDTM** [comment]
@DoctoreMad i mean they are still linked to the other bodies, blood volume is still body-specific

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**